### PR TITLE
Closes #6646 - Use app namespace for save login button tint

### LINKED ISSUE #6646: Save Login Button is Red on API 21/22
Need to use app namespace

### PR DESCRIPTION
**Pull request by ekager**
---

### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml [hunk 1]
```diff
@@ -121,12 +121,12 @@
         android:layout_height="wrap_content"
         android:layout_alignParentEnd="true"
         android:layout_marginTop="16dp"
-        android:backgroundTint="@color/button_state_list"
         android:text="@string/mozac_feature_prompt_save_confirmation"
         android:textAllCaps="false"
         android:textColor="?android:windowBackground"
         android:textSize="14sp"
         android:textStyle="bold"
+        app:backgroundTint="@color/button_state_list"
```
Comment (jonalmeida) [inline components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml:129]: Doesn't this require an `AppCompatButton`?

Comment (ekager) [reply components/feature/prompts/src/main/res/layout/mozac_feature_prompt_login_prompt.xml:129]: It seems to work in my testing. My understanding of the support library is that it does some auto-swapping at layout time Button for -> AppCompatButton etc, but I can be explicit in the declaration

**Comment by jonalmeida** [comment]
bors r-

I meant to approve only so you can bors merge if you were making changes in this PR.

**Comment by ekager** [comment]
Just tested on 21 + 22 without making it explicitly an AppCompatButton so I think this is fine :) 
catch me in a few hours being like jk we need appcompat lol

**Comment by ekager** [comment]
bors r=jonalmeida

**Pull request merged**